coordinator/src/models/hl7-v3/hl7-v3-datatypes-core.ts

```diff
@@ -173,6 +173,12 @@ export class Telecom implements ElementCompact {
   }
 
   constructor(use: TelecomUse, value: string) {
+    if (value) {
+      value = value.replace(/\s/g, "")
+      if (!value.startsWith("tel:")) {
+        value = `tel:${value}`
```

Comment:
Do we need to handle email, fax and textphone? PDS does, but maybe we can say these are unsupported for EPS?

Reply:
Not sure, I'll double check with Chris to see if it's required